openedx/stanford/cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -1,5 +1,23 @@
+import urllib
+
+from django.core.exceptions import PermissionDenied
 from django.core.urlresolvers import reverse
 
+from cms.djangoapps.contentstore.utils import reverse_library_url, reverse_usage_url
+from student.auth import has_studio_read_access
+from xmodule.modulestore.django import modulestore
+
+
+def get_course_and_check_access(course_key, user, depth=0):
```

Comment:
I don't like this much duplication, but it resolves some circular dependency issues.